testsuite/MDAnalysisTests/datafiles.py

```diff
@@ -38,13 +38,13 @@
 
 __all__ = [
     "PSF", "DCD", "CRD",  # CHARMM (AdK example, DIMS trajectory from JMB 2009 paper)
-    "DCD2", # CHARMM (AdK example, DIMS trajectory from PLOS Comput Biol paper)
+    "DCD2",  # CHARMM (AdK example, DIMS trajectory from PLOS Comput Biol paper)
```

Comment:
I would just ignore linting things for anything you aren't changing. Especially for this file given we're in the processing of changing it (see: #4056).